mods/ts/weapons/missiles.yaml

```diff
@@ -35,9 +35,9 @@ Bazooka:
 		ImpactSound: ssplash3.aud
 		ValidImpactTypes: Water
 	Warhead@4EffAir: CreateEffect
-		Explosion: small_twlt
-		ImpactSound: expnew12.aud
-		ValidImpactTypes: Air, AirHit
+		Explosion: tiny_twlt
+		ImpactSound: expnew05.aud
+		InvalidImpactTypes: Ground
```

Comment:
This doesn't look right to me, with the original setup, the explosion should only display when exploding in the air. With this change, it would play when hitting ground targets or anything on water as well.

If you say that's not the case, this appears to be a code-level bug.